### PR TITLE
Feature/testing w rtl

### PR DESCRIPTION
**Pull request by NewIncome**
Made required tests according to Microverse's Math-Magicians Testing Project

### CODE REVIEW

**Review by wathigo — CHANGES_REQUESTED** [review]
## Changes Required :o: 
Great job adding tests.
Let's make some changes before we move on.

### REQUIRED
* Use jest snapshots to make sure that all the elements in the Home, Calculator, and Quote pages have been displayed. 
Screenshot from 2021-01-16 06-57-22
* Include jest in devDependencies by running `npm install --save-dev jest`.

**After making the changes, submit another request for review.**

**Happy Coding :computer: 
Simon Wathigo**

**Comment by NewIncome** [comment]
> ## Changes Required o
> 
> Great job adding tests.
> Let's make some changes before we move on.
> ### REQUIRED
> 
>     * Use jest snapshots to make sure that all the elements in the Home, Calculator, and Quote pages have been displayed.
>       Screenshot from 2021-01-16 06-57-22
> 
>     * Include jest in devDependencies by running `npm install --save-dev jest`.
> 
> 
> **After making the changes, submit another request for review.**
> 
> **Happy Coding computer Simon Wathigo**

Hi @wathigo 
Please need some help.
I've been getting an error and can't manage to solve it. Regarding React tests.
I posted the seek for help in the Full-time student's Slack channel.

By the way, Jest was already in my devDependencies but anyway I ran that line to install it again.

File: src/components/App.js [hunk 1]
```diff
@@ -31,22 +30,27 @@ class App extends React.Component {
...
+      //           (total === null && next === null) ? '0' : next === null ? total : next
+      //           }
+      //         />
+      //       </div>
+      //       <div className="calcBtm">
+      //         <ButtonPanel onClick={this.handleClick} />
+      //       </div>
+      //     </div>
+      //   </div>
+      // </section>
```
Comment (bellom) [inline src/components/App.js]: It’s better to delete commented code. If you need to review a previous version of your code, use git history. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

**_Kindly remove before merging._**

**Review by bellom — APPROVED** [review]
## Hi @NewIncome  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**